core/plugins/org.polarsys.capella.core.preferences/src/org/polarsys/capella/core/commands/preferences/service/PropertyStore.java

```diff
@@ -172,18 +149,22 @@ private void writeProperties(ProjectScope project) throws IOException {
 
   /**
    * Convenience method to set a property
-   * @param name - the preference name
-   * @param value - the property value or null to delete the property
+   * 
+   * @param name
+   *          - the preference name
+   * @param value
+   *          - the property value or null to delete the property
    * @throws CoreException
    */
   private void setProperty(String name, String value) throws CoreException {
-    resource.setPersistentProperty(new QualifiedName(pageId, name), value);
```

Comment:
pageId is useless and make code unclear, we don't use it afterwards in ScopedCapellaProjectScope.getProjectValue